javascript/widget.js

```diff
@@ -106,7 +108,26 @@
                 (search == '' ? '' : '&q=' + search) +
                 '&offset=' + offset;
               return url;
-            }
+            },
+            buildSearchForm: buildSearchForm
+          },
+          trade_leads: {
+            title: 'Trade Leads',
+            resultTitleField: 'title',
+            displayFields: ['agency', 'topic', 'description', 'source', 'contract_value'],
+            searchUrl: function(search, offset) {
+              offset = offset || 0;
+              var url = host + '/trade_leads/search' +
+                '?api_key=' + apiKey +
+                '&offset=' + offset;
+              $.each(search, function(index, value) {
+                if (value != '') {
+                  url += '&' + index + '=' + value;
+                }
+              });
+              return url;
+            },
```

Comment:
trade_leads.searchUrl’s first argument is an object. The other searchUrl functions’ first arguments are string (the string to pass to the “q” field). I suggest you rework the other functions so that all searchUrl functions receive the same type of object as their first args. You may find that all three become identical.